src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/IVsTestConsoleWrapper.cs

```diff
@@ -4,6 +4,7 @@
 namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer.Interfaces
 {
     using System.Collections.Generic;
+    using System.Threading.Tasks;
```

Comment:
Should we create a `IVsTestConsoleWrapper2` for async methods? Async is probably the way forward for the API set. @Microsoft/vstest what does everybody think?

From implementation of the interface, probably the `VsTestConsoleWrapper` can implement both interfaces. We can possibly reduce the business logic duplication across async and non-async.